### PR TITLE
Apache HTTP Server content

### PR DESCRIPTION
**Pull request by vu-socprime**
## Proposed Changes

  - add Apache HTTP Server analytic rules
  - add Apache HTTP Server hunting queries
  - add Apache HTTP Server workbook

### CODE REVIEW

File: Solutions/ApacheHTTPServer/Analytic Rules/ApacheRequestToSensitiveFiles.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+id: d1c52578-3afc-11ec-8d3d-0242ac130003
+name: Apache - Request to sensetive files
```
Comment (NikTripathi) [inline Solutions/ApacheHTTPServer/Analytic Rules/ApacheRequestToSensitiveFiles.yaml:2]: Typo sensetive  -> sensitive

File: Solutions/ApacheHTTPServer/Analytic Rules/ApacheRequestToSensitiveFiles.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+id: d1c52578-3afc-11ec-8d3d-0242ac130003
+name: Apache - Request to sensetive files
+description: |
+  'Detects request to sensetive files.'
```
Comment (NikTripathi) [inline Solutions/ApacheHTTPServer/Analytic Rules/ApacheRequestToSensitiveFiles.yaml:4]: Typo sensetive  -> sensitive

**Review by NikTripathi — CHANGES_REQUESTED** [review]

**Comment by NikTripathi** [comment]
@aprakash13 Please review analytic rules. Thanks

File: Solutions/ApacheHTTPServer/Analytic Rules/ApacheRequestToSensitiveFiles.yaml (hunk 1 above)
Comment (vu-socprime) [reply Solutions/ApacheHTTPServer/Analytic Rules/ApacheRequestToSensitiveFiles.yaml]: fixed

File: Solutions/ApacheHTTPServer/Analytic Rules/ApacheRequestToSensitiveFiles.yaml (hunk 2 above)
Comment (vu-socprime) [reply Solutions/ApacheHTTPServer/Analytic Rules/ApacheRequestToSensitiveFiles.yaml]: fixed

**Review by NikTripathi — COMMENTED** [review]
Workbook looks good. Can merge after @aprakash13's approval.

**Comment by vmanojreddy** [comment]
@vu-socprime In  checks list Kqlvalidations failing due to  ApacheHTTPServer table does not refering to any table in CustomTables folder.
Path refrence https://github.com/Azure/Azure-Sentinel/tree/master/.script/tests/KqlvalidationsTests/CustomTables

**Comment by vu-socprime** [comment]
> @vu-socprime In checks list Kqlvalidations failing due to ApacheHTTPServer table does not refering to any table in CustomTables folder.
> Path refrence https://github.com/Azure/Azure-Sentinel/tree/master/.script/tests/KqlvalidationsTests/CustomTables

@vmanojreddy Updated. Thanks.

**Comment by NikTripathi** [comment]
@aprakash13 Please review analytic rules. Thanks

File: Solutions/ApacheHTTPServer/Analytic Rules/ApacheCVE-2021-41773.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+description: |
+  'Detects using Apache 2.4.49 flaw CVE-2021-41773 '
+severity: Medium
+requiredDataConnectors:
+  - connectorId: ApacheHTTPServer
+    dataTypes:
+      - ApacheHTTPServer
+tactics:
+  - InitialAccess
+  - Lateral Movement
```
Comment (aprakash13) [inline Solutions/ApacheHTTPServer/Analytic Rules/ApacheCVE-2021-41773.yaml:12]: LateralMovement should be one word without space.

File: Solutions/ApacheHTTPServer/Analytic Rules/ApacheMultipleServerErrorsRequestsFromSingleIP.yaml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+      - ApacheHTTPServer
+tactics:
+  - Impact
+  - InitialAccess
+relevantTechniques:
+  - T1498
+  - T1190
+  - T1133
+query: |
+  let threashold = 100;
```
Comment (aprakash13) [inline Solutions/ApacheHTTPServer/Analytic Rules/ApacheMultipleServerErrorsRequestsFromSingleIP.yaml:18]: Probably a small typo but if wanted we can change the spelling to 'threshold' here and in the query.

**Review by aprakash13 — CHANGES_REQUESTED** [review]
Overall, the queries look good. Some suggestions about things that we need to change:

•	For all the analytical rules, we need to add the version and the kind properties in the end – something like below: 
version: 1.0.0
kind: Scheduled

•	 Also, all the analytical rule generally also have the these 4 properties defined. Please put them as well in the analytical rules:
queryFrequency: 1d
queryPeriod: 7d
triggerOperator: gt
triggerThreshold: 0

  For more details about these see here:  https://github.com/Azure/Azure-Sentinel/wiki/Query-Style-Guide#queryfrequency

Also, before we can merge these, we need to resolve all the validation errors, that we currently have.

File: Solutions/ApacheHTTPServer/Analytic Rules/ApacheCVE-2021-41773.yaml (hunk 3 above)
Comment (vu-socprime) [reply Solutions/ApacheHTTPServer/Analytic Rules/ApacheCVE-2021-41773.yaml]: updated

**Comment by v-sabiraj** [comment]
Hey @vu-socprime, We have resolved the issue for PR Validations failure and request you to please update your PR branch with the latest changes from **Azure:master** so that the PR Validations can be executed.... Thanks!!!

**Comment by vu-socprime** [comment]
> Hey @vu-socprime, We have resolved the issue for PR Validations failure and request you to please update your PR branch with the latest changes from **Azure:master** so that the PR Validations can be executed.... Thanks!!!

@v-sabiraj Updated

**Comment by NikTripathi** [comment]
@aprakash13 Please approve the changes.

**Comment by spsocprime** [comment]
fixed

**Review by NikTripathi — APPROVED** [review]

**Pull request merged**